src/Microsoft.DocAsCode.Build.SchemaDriven/SchemaDrivenDocumentProcessor.cs

```diff
@@ -141,28 +141,25 @@ public override FileModel Load(FileAndType file, ImmutableDictionary<string, obj
                         {
                             LocalPathFromRoot = localPathFromRoot,
                         };
+                        fm.MarkdownFragmentsModel = new FileModel(
```

Comment:
why we still need create FragmentsModel when the content is null?

Reply:
This model is used to report a dependency. Even md doesn't exist, DocFx should think yaml has a dependency on it.